### DISCUSSION TITLE
Django deployment problem

### DISCUSSION THREAD

**Discussion by balogun14** [vercel/community#3099] [Help]
### Summary

i just deployed my django project and i am getting  status code 500 when i want to visit the site 

### Example

summary-bice.vercel.app

### Steps to Reproduce

i just uploaded my django code

**Comment by amyegan** [comment]
You should be able to get more detail from the deployment's [Runtime Logs](https://vercel.com/docs/concepts/observability/runtime-logs#). Your browser's developer tools console and network tab may have some helpful info as well. 

It may also be useful to wrap some of your code in a [try-except block](https://wiki.python.org/moin/HandlingExceptions) to print out extra details in the event of an error.
